Urban-Fun/View Controllers/ActivityDetailsViewController.m

```diff
@@ -104,14 +105,17 @@ - (void)setUpView {
     }
 }
 - (IBAction)didTapHost:(id)sender {
+    
     [self performSegueWithIdentifier:@"profileFromDetails" sender:sender];
 }
 
 
 - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
  if ([[segue identifier] isEqualToString:@"profileFromDetails"]){
      ProfileViewController *vc = [segue destinationViewController];
-     vc.profileToView = self.activity.host;
+     if (![self.activity.host.objectId isEqualToString:[User currentUser].objectId]){
+         vc.profileToView = self.activity.host;
```

Comment:
Is this correct? what's the logic behind this? seems like if the user of the activity is the host then you won't get `profileToView` in `ProfileViewController` which could potentially crash the app?

Reply:
Yes, this is correct because I have a conditional in the profile VC that says, if profileToView == nil, then profileToView = currentUser.